environment-dev.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - cartopy
   - geopandas
   - haversine
+  - importlib-metadata <8.0  # Required for xesmf >=8.4.0,<8.6 to work on Windows
```

Comment:
Is there a danger to pin this for everyone? 

Also, I don't know exactly why this is required for it to work on GitHub, but my local setup works with `importlib-metadata 8.4.0, esmpy 8.4.2, xesmf 0.8.7` without any issue...

Reply:
I grabbed this fix from here: https://github.com/CLIMADA-project/climada_petals/issues/132. 

I don't think it is dangerous, but from my understanding, the fact that the metadata is corrupted in `esmpy`/`esmf`, we need to have an older version here, otherwise this raises `ValueErrors` on import.